upload-s3/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Upload S3'
+description: 'Uploads package to S3'
+inputs:
+  bucket:
+    required: true
+  product:
+    required: true
+  branch:
+    required: true
+  build:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create and release
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/release.tar.gz --body release.tar.gz

```

Comment:
I think you need to add support for uploading the `release` folder instead of the tarball because some of our repos do not like tarball.
https://git.dev.d2l/projects/MIM/repos/bdp-ci/browse/deploy-s3.sh#81

Reply:
Ahh - I thought that was just for help while we were doing the cutover. Do you remember what repos had issues with the tarball? I'm thinking we can add support for `untar` in a future PR.

Reply:
I recall that I put a list of unsupported BDP/BEF repos in my previous Rally story's note section.

Reply:
Seems like its the system/perf tests for the BEF, and infrastructure & brightspace-saml-idp for BDP.